src/appmixer/intelswift/core/SaveSyncId/SaveSyncId.js

```
@@ -0,0 +1,29 @@
+'use strict';
+
+const { makeRequest } = require('../../commons');
+module.exports = {
+
+    async receive(context) {
+
+        const { platformType, contactId, tenantId, botId, origin, syncId } = context.messages.in.content;
+
+        let body = {};
+        if(!platformType) throw 'platformType not found';
```

ESLint

[error] Expected space(s) after "if".

(keyword-spacing)